misc/scripts/deploy-tools.sh

```diff
@@ -335,6 +351,14 @@ deployesx)
         fi
         deployesx
         ;;
+deployesxForUpgrade)
```

Comment:
I don't think it matters. The function name is long and to make it all lower case looks very ugly.